Source/WebKit/UIProcess/ios/DragDropInteractionState.h

```diff
@@ -78,7 +78,7 @@ class DragDropInteractionState {
     const DragSourceState& stagedDragSource() const { return m_stagedDragSource.value(); }
     enum class DidBecomeActive : bool { No, Yes };
     void clearStagedDragSource(DidBecomeActive = DidBecomeActive::No);
-    UITargetedDragPreview *previewForLifting(UIDragItem *, UIView *contentView, UIView *previewContainer, const std::optional<WebCore::TextIndicatorData>&) const;
+    UITargetedDragPreview *previewForLifting(UIDragItem *, UIView *contentView, UIView *previewContainer, const RefPtr<WebCore::TextIndicator>&&) const;
```

Comment:
I think having a const rvalue ref interface provides an unexpected restriction on lvalues? We should probably just:
```suggestion
    UITargetedDragPreview *previewForLifting(UIDragItem *, UIView *contentView, UIView *previewContainer, RefPtr<WebCore::TextIndicator>&&) const;
```

Reply:
I guess we already committed prior art in `deliverDelayedDropPreview()`, though it shouldn't hurt if we don't follow suit here!